superbench/runner/runner.py

```diff
@@ -102,12 +102,13 @@ def __get_enabled_benchmarks(self):
                 return list(self._sb_config.superbench.enable)
         return [k for k, v in self._sb_benchmarks.items() if v.enable]
 
-    def __get_mode_command(self, benchmark_name, mode):
+    def __get_mode_command(self, benchmark_name, mode, timeout=None):
```

Comment:
change default value to 0?

Reply:
If we don't set the timeout in config file, it will be `None`. I think use `None` as default is better?